### DISCUSSION TITLE
Daily forecasted high and low temperature from API?

### DISCUSSION THREAD

**Discussion by mattdm** [weather-gov/api#523] [Q&A]
It's pretty typical for weather forecasts for the coming week (or whatever) to show the predicted highs and lows for each day. The weather-gov API does not seem to include this. Am I missing something, or is this by design?

Is there a better approach than getting the hour weather forecast and taking the min and max for hours within each day?

**Comment by jah1007** [comment]
This is available in the raw data, example https://api.weather.gov/gridpoints/BUF/27,8. There are two parameters maxTemperature and minTemperature which I think provide the information you are seeking.

**Reply by mattdm** [reply]
Perfect, thanks! I was looking in  (e.g.) https://api.weather.gov/gridpoints/BUF/27,8/forecast.

**Reply by scadergit** [reply]
The current /forecast endpoint schema was simply copied from the Forecast application, and we all look forward to the opportunity to update it with this and other common elements like chance of precipitation.

**Reply by rparanson** [reply]
Thank you.  I see max and min temps for specific time periods but not for the full day.  Also, these appear to only be in Celcius, not Fahrenheit.  Any guidance would be appreciated for this new user.  Thanks!
